packages/components/psammead-copyright/package.json

```diff
@@ -20,15 +20,18 @@
     "@bbc/gel-foundations": "^0.3.0",
     "@bbc/psammead-styles": "^0.3.2"
   },
+  "devDependencies": {
+    "@bbc/psammead-test-helpers": "^0.3.3",
+    "@bbc/psammead-visually-hidden-text": "^0.1.8"
```

Comment:
Looks like we're missing `styled-components` from our `devDependencies` here.

Reply:
good spot